mlrun/config.py

```diff
@@ -545,13 +545,20 @@ def get_storage_auto_mount_params():
 
         return auto_mount_params
 
-    def get_default_function_pod_resources(self):
+    def get_default_function_pod_resources(
+        self, with_gpu_requests=True, with_gpu_limits=True
```

Comment:
Besides testing purposes, you only ever call this function with `(False, False)`.
Then why set the arguments' defaults to `True` and not `False`?